src/frontend/components/usa--breadcrumb.tsx

```diff
@@ -16,13 +17,15 @@ export function USABreadcrumb({activeItem, links}: USABreadcrumbProps) {
           <span>Home</span>
         </BreadcrumbLink>
       </Breadcrumb>
-      {links.map((link) => (
```

Comment:
I added a comment to the top of the file so we would remember why this is here in the future, but the BreadcrumbBar type failed because it wanted only an array of elements, adding the `map` function inline did not meet that requirement. Wrapping it in the empty `<>` fixes the issue without changing how it is shown on the frontend.